deep_speech_2/decoder.py

```diff
@@ -36,25 +38,164 @@ def ctc_best_path_decode(probs_seq, vocabulary):
     return ''.join([vocabulary[index] for index in index_list])
 
 
-def ctc_decode(probs_seq, vocabulary, method):
+class Scorer(object):
```

Comment:
If more language models are evolved, the Scorer will be redesigned. Temporarily we use one class to avoid redundancy.